Packs/CiscoASA/Integrations/CiscoASA/CiscoASA.py

```diff
@@ -32,9 +33,13 @@ def login(self, isASAv) -> None:
             self._headers['X-Auth-Token'] = auth_token
             self.auth_token = auth_token
 
-    def __del__(self):
-        if self.isASAv and self.auth_token:
-            self._http_request('DELETE', f'/api/tokenservices/{self.auth_token}', resp_type='response')
+    def logoff(self):
+        try:
+            if self.isASAv and self.auth_token:
+                self._http_request('DELETE', f'/api/tokenservices/{self.auth_token}', resp_type='response')
+        except Exception as e:
+            # if failed to logoof just write to log. no need to raise error
+            LOG(f'Logoff error: {str(e)}')
```

Comment:
switch to `demisto.debug` or add `LOG.print_log()` right after.
since the call to logout is after the `return_error` call